### ISSUE TITLE
aardvark logout not working

### ISSUE THREAD

**Issue by pekeler** [arangodb/arangodb#1262]
arangod.conf:
```
[server]
authenticate-system-only = yes
disable-authentication = no
```
but Logout doesn't seem to clear the hot-auth credentials, i.e. once logged in, always logged in. (tested on v2.4.4 running in production mode, in Safari and Firefox).

**Comment by fceller** [comment]
Currently we are using HTTP Authentication, which does not logout reliable.
We are currently working on a better solution. Please stay tuned.

**Comment by pluma** [comment]
Fixed via #1268.
